screenshot-iframe.js

```diff
@@ -0,0 +1,80 @@
+import {storageHost} from './constants';
+import url from 'url';
+
+function parseQuery(queryString) {
+    const query = {};
+    const pairs = (queryString[0] === '?' ? queryString.substr(1) : queryString).split('&');
+    for (let i = 0; i < pairs.length; i++) {
+      const pair = pairs[i].split('=');
+      const k = decodeURIComponent(pair[0]);
+      if (k) {
+        const v = decodeURIComponent(pair[1] || '');
+        query[k] = v;
+      }
+    }
+    return query;
+}
+
+export const getScreenshotBlob = async (reqUrl, hash, ext, type, width, height) => {
+
+    var iframeExists = document.getElementsByName('screenshotIframe');
+
+    if(!iframeExists.length) {
+        var iframe = document.createElement('iframe');
+        iframe.width = '0px';
+        iframe.height = '0px';
+        iframe.name = 'screenshotIframe';
+        document.body.appendChild(iframe);
+    }
+
+    if(hash) {
+        var tempUrl = `https://app.webaverse.com/${hash}.${ext}/preview.${type}`
+    }
+    else {
+        var tempUrl = `https://app.webaverse.com/screenshot.html?url=${reqUrl}&hash=${hash}&ext=${ext}&type=${type}&width=${width}&height=${height}`;
+    }
+
+    const u = url.parse(tempUrl, true);
+
+    const spec = (() => {
```

Comment:
what is the use of the function here ?

Can't we construct the url directly ?

Reply:
Yes we can, fixed it now.